prebid-server/endpoints/openrtb2/pbs-endpoint-auction.md

```diff
@@ -110,14 +110,49 @@ This is a corresponding sample response to a sample OpenRTB 2.5 bid request:
+| source.tid | 2.5 | if source.tid is not set:
+   set source.tid to a random UUID
+if host config generate-storedrequest-bidrequest-id config is true
+    if the storedrequest is from AMP or from a top-level stored request (ext.prebid.storedrequest), then replace any existing $.source.tid with a random value
+if $.source.tid contains "{{UUID}}", replace that macro with a random value |
+| imp[].id | 2.5 | if host config generate-storedrequest-bidrequest-id config is true
+    if any $.imp[].id is missing, set it to a random 16-digit string. (Note: this wasn't in issue 1507)
+    if the storedrequest is from AMP **or** from a top-level stored request (ext.prebid.storedrequest), confirm that all $.imp[].id fields in the request are different. If not different, re-number them all starting from "1".
+ |
+| imp[].ext.tid | 2.x | if imp[n].ext.tid is not set:
+       set imp[n].ext.tid to a randomly generated UUID
+   if host config generate-storedrequest-bidrequest-id config is true
+       if the storedrequest is from AMP or from ext.prebid.storedrequest, then replace any existing $.imp[n].ext.tid with a random value
+  if $.imp[n].ext.tid contains "{{UUID}}", replace that macro with a random value |
+| cur | 2.5 | Supports either a string or an array. If array, the first element is taken to be the "Ad Server Currency" for purposes of [currency conversion](/prebid-server/features/pbs-currency.html).
+| exp | 2.5 | See the [expiration](#expiration) section below |
+| tmax | 2.5 | See the [timeout](#timeout) section below |
+| device.lmt | 2.5 | See special processing for iOS apps defined in [issue 1699](https://github.com/prebid/prebid-server/issues/1699) |
+| regs.gdpr | 2.6.202210 | Bidders supporting 2.5 only: downgraded to regs.ext.gdpr | 
+| regs.us_privacy | 2.6.202210 | Bidders supporting 2.5 only: downgraded to regs.ext.us_privacy |
+| user.consent | 2.6.202210 | Bidders supporting 2.5 only: downgraded to user.ext.consent |
+| imp.rwdd | 2.6.202210 | Bidders supporting 2.5 only: downgraded to imp.ext.prebid.is_rewarded_inventory (PBS-Java) or removed (PBS-Go) |
+| user.eids | 2.6.202210 | Bidders supporting 2.5 only: downgraded to user.ext.eids |
+| source.schain | 2.6.202210 | Bidders supporting 2.5 only: downgraded to source.ext.schain |
+| wlangb, {content, device}.langb, cattax, {site, app, publisher, content, producer}.cattax, ssai, {app, site}.content.{network, channel}, {app, content, site, user}.kwarray, device.sua | 2.6.202210 | Bidders supporting 2.5 only: these fields are removed |
+| {video, audio}.{rqddurs, maxseq, poddur, podid, podseq, mincpmpersec, slotinpod} | 2.6.202210 | not yet supported by PBS |
+| regs.gpp | 2.6.202211 | Bidders supporting 2.5 only: this field is removed |
+| regs.gpp_sid | 2.6.202211 | Bidders supporting 2.5 only: this field is removed |
+| dooh | 2.6.202211 | not yet supported by PBS |
+| imp.qty | 2.6.202211 | not yet supported by PBS |
+| imp.dt | 2.6.202211 | not yet supported by PBS |
```

Comment:
No action is taken on `qty` and `dt` yet in PBS-Go, but they exist in the schema and will be forwarded to adapters supporting 2.6 and removed for adapters supporting 2.5 only.